entrypoint.sh

```diff
@@ -97,25 +111,27 @@ populating_secret() {
   done
   unset IFS
 
-  if [ "$INPUT_EXPORT_ENV" == "true" ]; then
-    # To support multiline secrets, we'll use the heredoc syntax to populate the environment variables.
-    # As the heredoc identifier, we'll use a randomly generated 64-character string,
-    # so that collisions are practically impossible.
-    random_heredoc_identifier=$(openssl rand -hex 32)
+  # To support multiline secrets, we'll use the heredoc syntax to populate the environment variables.
+  # As the heredoc identifier, we'll use a randomly generated 64-character string,
+  # so that collisions are practically impossible.
+  delimiter="$(openssl rand -hex 32)"
```

Comment:
Appreciate the comments and the explanation on the call 🙏

Do you think it'd be worth linking to the [Multiline string](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings) section of the GitHub Docs here?

Reply:
Added the link with [f195e2d](https://github.com/1Password/load-secrets-action/pull/49/commits/f195e2df03423d2e879db28f05869acaf88aa6b3).